src/utils/borrowPools/mainnetPools.js

```diff
@@ -1389,7 +1389,7 @@ export default [
     contractChain: 1,
     id: 28,
     stabilityFee: 7,
-    interest: 0,
```

Comment:
Maybe try to find a way to not hardcode value like that here and read it onchain?